html.md

```diff
@@ -167,3 +167,27 @@ List the entries like in a dictionary or encyclopedia.
    Password: <input type = "password" name = "password" /> <br> <!-- Single line password input-->
  </form>
 ```
+
+## HTML Favicon
```

Comment:
We want to cover basics only in Cheat Sheets, this is advanced information. You can always create these as posts in OneCompiler